internal/cli/command_factory.go

```diff
@@ -111,7 +114,7 @@ func (factory *CommandFactory) Build(command CommandDefinition) *cobra.Command {
 				factory.profile.ClearSession()
 				profileErr := factory.profile.Save()
 				if profileErr != nil {
-					factory.errLogger.Fatal(fmt.Errorf("failed to clear session: %s", profileErr.Error()))
+					factory.warnLogger.Printf("failed to clear session: %s", profileErr.Error())
```

Comment:
we should use `factory.ui.Print(terminal.NewTextLog("..."))` here.  the one catch is we still don't have a WARN level severity, which you could add to the `terminal` package and maybe create a `terminal.NewWarning` function (which does the same thing as NewTextLog but uses a different LogLevel

Reply:
Done! I added a nolint line over the factory print to ignore the error. Wasn't sure of a cleaner alternative since we want to prioritize the caught upstream error in this scenario